### DISCUSSION TITLE
Install Error and cant view alert in wweb interface

### DISCUSSION THREAD

**Discussion by Cavoi1117** [Security-Onion-Solutions/securityonion#11433] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

500

### Storage for /nsm

328

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

hi, When I install manager by using ISO in EXSI, I got a error like this:
![viber_image_2023-09-26_16-26-58-586](https://github.com/Security-Onion-Solutions/securityonion/assets/80514435/ffdf6344-d582-4165-8043-6d965333d63b)
and I do exactly like this picture show, I got this error:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/80514435/83116ec8-4143-4a13-ae0e-11c464cea0d8)
and after I reboot manager machine, I can connect to the web interface with no problem. But, when I go to the alert session, I got this error
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/80514435/deed0509-6565-40da-b89d-7ac3d6de487d)
how can I fix this problem, thanks for answer

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
First, please note that when you install a manager node for a distributed deployment, you most likely won't have any alerts until you join additional nodes to the deployment. For example, for NIDS alerts you would need a forward node to generate network alerts and a search node to store those alerts.

As for the installation error, here are a few things to look into:
- 16GB RAM would be the bare minimum. You might try increasing if possible.
- What is the underlying storage in your ESXi? NVME, SSD, rotating platters? Is the storage local or remote?
- Check the log files in /opt/so/log/ for additional clues.

Finally, here is some additional documentation to review:
https://docs.securityonion.net/en/2.4/best-practices.html
https://docs.securityonion.net/en/2.4/architecture.html
https://docs.securityonion.net/en/2.4/hardware.html
https://docs.securityonion.net/en/2.4/help.html
